.scrutinizer.yml

```diff
@@ -22,5 +27,5 @@ build:
             -
                 command: 'gulp phpunit-codecoverage'
                 coverage:
-                    file: 'build/logs/clover.xml'
+                    file: 'cover.clover'
```

Comment:
I believe this is misspelled? however, the whole `build` section can be removed now that it's working on Travis. Let's do that and confirm the external code coverage is still working.